jb.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Please replace with `#!/usr/bin/env bash` (see previous discussion).

Reply:
This whole file will go away in favor of `Makefile.dune`.